tests/pipeline/test_blaze.py

```diff
@@ -843,10 +849,21 @@ def compute(self, today, assets, out, data):
             check_dtype=False,
         )
 
-    @with_extra_sid
-    def test_deltas(self, asset_info):
-        expr = bz.Data(self.df, name='expr', dshape=self.dshape)
-        deltas = bz.Data(self.df, dshape=self.dshape)
+    @with_ignore_sid
+    def test_deltas(self, asset_info, extra_sid_to_ignore):
+        if not extra_sid_to_ignore:
+            df = self.df
+        else:
+            dates = self.dates.repeat(4)
+            df = pd.DataFrame({
```

Comment:
we shouldn't duplicate `self.df` here because it could change. We could just add the data for `E` to a copy of `self.df`